client/GameBoard.jsx

```diff
@@ -109,9 +113,26 @@ class InnerGameBoard extends React.Component {
     }
 
     onPlotDeckClick() {
+        if (this.state.showUsedPlotDeck) { this.setState({ showUsedPlotDeck: !this.state.showUsedPlotDeck }); }
```

Comment:
This will fail the eslinting rules, block statements should be on their own lines